daemon/response.go

```diff
@@ -27,8 +27,11 @@ import (
 	"path/filepath"
 	"strconv"
 
+	"github.com/gorilla/websocket"
```

Comment:
Package is `golang-websocket-dev`, and yes we need it added to `debian/control`. Also `dependencies.tsv` needs updating, ideally pointing at the same one we'd get from `golang-websocket-dev/xenial`.